perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -1389,6 +1389,27 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Deletes multiple attribute definitions from Perun.
+	 *
+	 * Deletion fails if any entity in Perun has
+	 * any value for any of these attributes set.
+	 *
+	 * @param attributes List<int> AttributeDefinition <code>id</code>
```

Comment:
This method actually deletes AttributeDefinition including all values set for any entity. Can you please update javadoc accordingly and fix the same docs for deleteAttribute() ? Also in GUI please include big warning about it. Perun admin must know what he does. Thank you.

Reply:
@zlamalp Done